README.md

```diff
@@ -882,6 +882,10 @@ Justin Whear wrote a Zig wrapper available at https://github.com/jwhear/roaring-
 
 https://groups.google.com/forum/#!forum/roaring-bitmaps
 
+# Contributing
+
+When contributing a change to the project, please run `clang-format` using Clang Format version 17 after making any changes. If Clang Format 17 is installed, this can also be accomplished by running the formatting script in `tools/clang-format.sh`. A github action runs on all PRs to ensure formatting is consistent with this.
```

Comment:
This could be updated now that you don't need to have clang-format 17 installed locally if you have docker